lib/ld-key-pair.js

```diff
@@ -302,13 +308,14 @@ class Ed25519KeyPair extends LDKeyPair {
   fingerprint() {
     const buffer = new forge.util.createBuffer();
 
-    // ed25519 cryptonyms are multiformat encoded values, specifically:
-    // (multicodec 0x30 + ed25519-pub 0xed + key bytes)
-    const pubkeyBytes = forge.util.binary.base58.decode(this.publicKeyBase58);
-    buffer.putBytes(forge.util.hexToBytes('30ed'));
+    // ed25519 cryptonyms are multicodec encoded values, specifically:
+    // (multicodec ed25519-pub 0xed01 + key bytes)
+    const pubkeyBytes = base58.decode(this.publicKeyBase58);
+    buffer.putBytes(forge.util.hexToBytes('ed01'));
```

Comment:
Not sure if this sort of micro-optimization matters (or if it even optimizes) but I think you could have done this instead here:

```
buffer.putByte(0xed);
buffer.putByte(0x01);
```